src/client/package.json

```diff
@@ -22,7 +22,8 @@
   "scripts": {
     "clean": "rm -rf dist",
     "start": "npm run dev",
-    "lint": "eslint src",
+    "dev:local": "npm run dev -- --endpoint local",
```

Comment:
Yeah I mean some config.  It's non-obvious how the "webpack magic" works.
Just a suggestion though.

On 13 April 2016 at 15:48, Keith Woods <notifications@github.com> wrote:

> In src/client/package.json
> <https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/240#discussion_r59561864>
> :
>
> > @@ -22,7 +22,8 @@
> >    "scripts": {
> >      "clean": "rm -rf dist",
> >      "start": "npm run dev",
> > -    "lint": "eslint src",
> > +    "dev:local": "npm run dev -- --endpoint local",
>
> what do you mean config we pass? Do you mean a file on disk, like
> environment-config.json?
>
> Ben did suggest this, we thought of just hardcoding it with a comment. I
> guess we could have a single file with 1 prop in there for url.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/240/files/676a00b7d166d210645422604e28d23154274a24#r59561864>
>

-- 
 

*This email and any attachments to it may be confidential and are intended 
solely for the use of the individual to whom it is addressed. Any views or 
opinions expressed are solely those of the author and do not necessarily 
represent those of Adaptive Financial Consulting Ltd. *

*If you are not the intended recipient of this email, you must neither take 
any action based upon its contents, not copy nor show it to anyone.*

*Please contact the sender if you believe you have received this email in 
error.*

*Adaptive Financial Consulting Ltd is a company registered in England and 
Wales. Registered number: 08105955. Registered office: 2nd floor, 11 
Leadenhall Street, London EC3 V1LP.*